iree/compiler/Conversion/LinalgToSPIRV/ConcretizeTileAmongWorkgroupsPass.cpp

```diff
@@ -463,6 +463,12 @@ class ConcretizeTileAmongWorkgroupsPass
       llvm::dbgs() << "\n";
     });
 
+    // Make sure we don't have more than 3 dimensions.
```

Comment:
It would be better to rebase this on https://github.com/google/iree/pull/5283. You might not need this after that.

Reply:
For now will leave this as-is. Would like to land this soon if possible so we can have a checkpoint for evaluating performance impacts of upcoming refactorings.